src/contracts/facilitators/gsm/interfaces/IGhoReserve.sol

```diff
@@ -0,0 +1,111 @@
+// SPDX-License-Identifier: MIT
+pragma solidity ^0.8.0;
+
+/**
+ * @title IGhoReserve
+ * @author Aave/TokenLogic
+ * @notice Defines the behaviour of a GhoReserve
+ */
+interface IGhoReserve {
+  /**
+   * @dev Struct data representing GHO usage.
+   * @param limit The maximum amount of GHO that can be used
+   * @param used The current amount of GHO used
+   */
+  struct GhoUsage {
+    uint128 limit;
+    uint128 used;
+  }
+
+  /**
+   * @dev Emitted when GHO is transferred from the reserve to an entity
+   * @param entity The address receiving the GHO tokens
+   * @param amount The amount of token to transfer
+   */
+  event GhoRestored(address indexed entity, uint256 amount);
+
+  /**
+   * @dev Emitted when GHO is transferred from the reserve
+   * @param to The address receiving the GHO tokens
+   * @param amount The amount of token to transfer
+   */
+  event GhoTransferred(address indexed to, uint256 amount);
+
+  /**
+   * @dev Emitted when GHO is restored to the reserve by an entity
+   * @param entity The address restoring the GHO tokens
+   * @param amount The amount of token restored
+   */
+  event GhoUsed(address indexed entity, uint256 amount);
```

Comment:
cosmetics nit, but i d move this up so it's easier to read: used, restored and transferred in order